Code/Framework/AzFramework/AzFramework/Physics/ShapeConfiguration.cpp

```diff
@@ -72,8 +77,9 @@ namespace Physics
         }
     }
 
-    SphereShapeConfiguration::SphereShapeConfiguration(float radius)
+    SphereShapeConfiguration::SphereShapeConfiguration(float radius, const AZ::Vector3& scale)
         : m_radius(radius)
+        , ShapeConfiguration(scale)
```

Comment:
[nit] normally the parent class input goes first because that's the order it'll happen. Having it at the end looks like the order it'll happen is different, so it's preferred to put it first.

Same in the other classes.